src/main/java/mate/academy/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(
+        @NotBlank
+        @Length(min = 8, max = 35)
+        @Email
+        String email,
+        @NotBlank
```

Comment:
Should you validate the length?

Reply:
I think no, it's not necessary here. I've removed it.